_pages/documentation.md

```diff
@@ -19,10 +19,12 @@ flow:
 Information and documentation on the bootloader is stored within the source. For more information 
 in the source, here are some pointers:
 
-boot/bootutil: The core of the bootloader itself.
-boot/boot_serial: Support for serial upgrade within the bootloader itself.
-boot/zephyr: Port of the bootloader to Zephyr
-boot/mynewt: Mynewt bootloader app
-boot/mbed: Port of the bootloader to Mbed-OS
-imgtool: A tool to securely sign firmware images for booting by MCUboot.
-sim: A bootloader simulator for testing and regression
\ No newline at end of file
+boot/bootutil: The core of the bootloader itself. \
+boot/boot_serial: Support for serial upgrade within the bootloader itself. \
+boot/zephyr: Port of the bootloader to Zephyr \
+boot/mynewt: Mynewt bootloader app \
+boot/mbed: Port of the bootloader to Mbed-OS \
+boot/espressif: Port of the bootloader to Espressif IDF \
+boot/nuttx: Port of the bootloader to NuttX \
```

Comment:
```suggestion
boot/nuttx: Bootloader application and port of MCUboot interfaces for NuttX. \
boot/espressif: Bootloader application and MCUboot port for Espressif SoCs. \
```
Here it might be interesting to follow the same order and use the same text from the README.md file of the mcuboot repo.